agent/agent.go

```diff
@@ -263,14 +259,9 @@ func (a *Agent) resetLogging(ctx context.Context, tc *taskContext) error {
 	}
 
 	sender, err := GetSender(ctx, a.opts.LogPrefix, tc.task.ID)
-	if err != nil {
-		return errors.Wrap(err, "problem getting sender")
-	}
-	err = grip.SetSender(sender)
-	if err != nil {
-		return errors.Wrap(err, "problem setting sender")
-	}
-	return nil
+	grip.Error(errors.Wrap(err, "problem getting sender"))
+
+	grip.Error(errors.Wrap(grip.SetSender(sender), "problem setting sender"))
```

Comment:
should some of these things become alerts?

Reply:
not yet since these errors happen quite often (EVG-5831)